src/MLJGLMInterface.jl

```diff
@@ -327,4 +327,301 @@ metadata_model(
     path = "$PKG.LinearCountRegressor"
 )
 
+"""
+$(MMI.doc_header(LinearRegressor))
+
+`LinearRegressor` assumes the target is a continuous variable
+whose conditional distribution is normal with constant variance, and whose
+expected value is a linear combination of the features (identity link function).
+Options exist to specify an intercept or offset feature.
+
+# Training data
+
+In MLJ or MLJBase, bind an instance `model` to data with
+
+    mach = machine(model, X, y)
+
+Where
+
+- `X`: is any table of input features (eg, a `DataFrame`) whose columns
+  are of scitype `Continuous`; check the scitype with `schema(X)`
+
+- `y`: is the target, which can be any `AbstractVector` whose element
+  scitype is `Continuous`; check the scitype with `schema(y)`
+
```

Comment:
There's a line I'd like to insert here on how to train the machine. Such a line should go at the end of the "Training data" section for every model docstring. Here it is in MLJDecisionTreeInterface.jl: https://github.com/JuliaAI/MLJDecisionTreeInterface.jl/blob/274b2a907d39a5e8b238f5a7d679afd9dadf3bc1/src/MLJDecisionTreeInterface.jl#L342

Reply:
done!